openshift/openshiftstack/application/openshift-service-mesh/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: istio-system
```

Comment:
would it be possible to have a custom namespace instead of istio-system? Like a prefix for the namespace with the application name, kubeflow-istio-system? Our envisioned model on a shared cluster is  eine having one service mesh control plane per application, where an application can have components in multiple namespaces / projects. So, single cluster with multiple separated meshes and control planes. One of those meshes is thought to be for Kubeflow.